documentation/smg/08_setting_up_jenkins_ci.md

```diff
@@ -27,6 +27,7 @@ For pull requests:
   occurs.
 - When Jenkins receives a notification from GitHub it will checkout out the
   relevant pull request branch and merge it with master.
+- PR labels are checked to determine CI process
```

Comment:
do we need to discuss the processing of `Herbert_<branchname>` labels in ths document?